content/en/docs/languages/python/getting-started.md

```diff
@@ -640,14 +640,11 @@ emitted to the console, with separate counts for each roll value:
 
 ## Send telemetry to an OpenTelemetry Collector
 
-The [OpenTelemetry Collector](/docs/collector/getting-started/) is a critical
-component of most production deployments. Some examples of when it's beneficial
-to use a collector:
+The [OpenTelemetry Collector](/docs/collector/getting-started/) is a critical component of most production deployments. Some examples of when it's beneficial to use a collector include:
 
-- A single telemetry sink shared by multiple services, to reduce overhead of
-  switching exporters
+- Creating a single telemetry sink shared by multiple services, to reduce overhead of switching exporters
 - Aggregating traces across multiple services, running on multiple hosts
-- A central place to process traces prior to exporting them to a backend
+- Creating a central place to process traces prior to exporting them to a backend
```

Comment:
the items of that list are related to "when it's beneficial to use a collector", so I think this should be something like "if you need...", not entirely sure, cc @open-telemetry/docs-approvers

Reply:
Ahh I see what you are saying.

In that case, how does the following phrasing sound?

"Some examples of when it's beneficial to use a collector include when you need to:

- Create a single telemetry sink shared by multiple services, to reduce overhead of switching exporters
- Aggregate traces across multiple services, running on multiple hosts
- Process your traces in a central place prior to exporting them to a backend"

Reply:
yes, that's better, so works for me 👍

Reply:
@astanley-work can you apply those changes?